content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,123 @@
+# data status
+
+Show changes in data tracked by DVC between the last commit, the dvcfiles and
+the workspace.
```

Comment:
```suggestion
Show changes in data tracked by DVC between the last committed project
version (Git `HEAD`), its [metafiles] and the <abbr>workspace</abbr>.

[metafiles]: /doc/user-guide/project-structure/dvcyaml-files
```

Reply:
"last committed project version (Git `HEAD`)" is too verbose.

I think "the last Git commit" is enough. Also "its metafiles" is not clear, whose metafiles?

Reply:
Regarding `abbr`, it's in the description, we don't need to add it everywhere.

Reply:
> Regarding abbr, it's in the description, we don't need to add it everywhere.

The existing pattern is to put it in the intro and then skip in the description but add it again every so many paragraphs or once per section.

Reply:
Also, it was not in the description at the time, see https://github.com/iterative/dvc.org/pull/3812#discussion_r936287875. And I never said we should "add it everywhere".